e2e/tests/e2e/WorkspaceCreationAndLsInitialization.spec.ts

```diff
@@ -45,7 +45,9 @@ suite('E2E', async () => {
         });
 
         test('Create and open workspace', async () => {
-            await newWorkspace.createAndRunWorkspace(namespace, workspaceName, 'Java Maven', sampleName);
+            await newWorkspace.typeWorkspaceName(workspaceName);
```

Comment:
I guess, that instead of this change, we should change https://github.com/eclipse/che/blob/master/e2e/pageobjects/dashboard/NewWorkspace.ts#L192 (remove the logic of selecting sample)